auto_backup/data/backup_data.yml

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```

Comment:
@pedrobaeza do you have any pointer about the fact they are removed in v10? or deprecated in v9? May be we should discuss it at the OCA... I think it's a bit late to require no YAML test anymore in v9 given the OCA is already late on that v9 transition...